### ISSUE TITLE
[Theme Request] Yatogami Tohka

### ISSUE THREAD

**Issue by EdgarsFeic69** [doki-theme/doki-master-theme#76]
## Who/What do you want to be featured?

<p>Tohka Yatogami from Date a Live</p>

### What do they look like?

<img src="https://images3.alphacoders.com/672/672163.jpg" alt="">
<img src="https://images7.alphacoders.com/101/1014209.png" alt="">
<img src="https://images3.alphacoders.com/966/966664.png" alt="">

## What type of theme?

- [ ] Dark Theme

## Additional Information (Optional)

  
<p>She's one of the best waifus from Date A Live, well, they are all good</p>

**Comment by Unthrottled** [comment]
> She's one of the best waifus from Date A Live, well, they are all good

Yeah, I'm quite partial to Kurumi's aesthetics if I do say so myself.

![kurumi](https://i.pinimg.com/736x/4b/5f/4c/4b5f4c19dbe877ae1d2d5936e35af7a3.jpg)

There are quite a few characters in this series that I've seen floating around and didn't know there names, thanks for bringing this to my attention! I'll keep her in mind if I need another dark purple theme.
